src/tokens/e2eTokens.ts

```diff
@@ -1,9 +1,9 @@
 import { StoredTokenBalances } from 'src/tokens/slice'
 
 // alfajores addresses
-const cUSD = '0x874069Fa1Eb16D44d622F2e0Ca25eeA172369bC1'
-const cEUR = '0x10c892A6EC43a53E45D0B916B4b7D383B1b78C0F'
-const CELO = '0xF194afDf50B03e69Bd7D057c1Aa9e10c9954E4C9'
+const cUSD = '0x874069Fa1Eb16D44d622F2e0Ca25eeA172369bC1'.toLowerCase()
+const cEUR = '0x10c892A6EC43a53E45D0B916B4b7D383B1b78C0F'.toLowerCase()
+const CELO = '0xF194afDf50B03e69Bd7D057c1Aa9e10c9954E4C9'.toLowerCase()
```

Comment:
These changes were required to fix an e2e test that was failing to pass, though I'm still not quite sure why. TLDR, since these addresses were not normalized, they [were being stored in redux](https://github.com/valora-inc/wallet/blob/main/src/tokens/saga.ts#L266) with non-normalized keys. The `tokenAddress` fields returned in responses from blockchain-api, however, _are_ lowercase, which means that the `useTokenInfo` hook [here](https://github.com/valora-inc/wallet/blob/main/src/tokens/hooks.ts#L12) was failing to find the tokens, since the e2e tokens were placed into redux with non-normalized token addresses for keys.

From looking at blockchain-api, I don't believe that we ever switched from using non-normalized to normalized strings for token addresses at any point; I believe they've always been lowercased, so I'm not certain why this issue is appearing in this PR.